text/0000-cbool.md

```diff
@@ -0,0 +1,28 @@
+- Feature Name: cbool
+- Start Date: Mon Mar  9 00:16:53 CET 2015
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Specify that `bool` is compatible with the `_Bool` C type.
+
+# Motivation
+
+You cannot safely call ffi functions with boolean arguments without a compatible type.
+
+# Detailed design
+
+Specify that `bool` is compatible with the `_Bool` C type and inform the compiler that `bool` is ffi safe.
+
+# Drawbacks
+
+None right now. This should be a no-op on all supported platforms.
```

Comment:
What's the plan for platforms that don't use tow's complement? If there is no such plan, then why would you worry more about the representation of `bool` than the representation of `u8`?